### PR TITLE
Seeders

### PR DESCRIPTION
**Pull request by ersel**

### CODE REVIEW

**Review by kirakoki — COMMENTED** [review]
Is this PR ready to be merged?

**Comment by ersel** [comment]
@kirakoki not yet, I don't think it actually generates the `INSERT` queries yet. I need to spend a few more hours on it.

File: cli/recipes/seed/utils/getFakerExample.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  return string;
+};
+
+const getFakerExample = (methodToCall) => {
+  try {
+    const out = getFakerValue(methodToCall);
+    return trimSampleOutput(out);
+  } catch (error) {
+    // console.error("CLI failed to generate seed data.");
+    // console.log(error);
```
Comment (kirakoki) [inline cli/recipes/seed/utils/getFakerExample.js:18]: do we need to notify the user?

File: cli/recipes/seed/index.js [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+                `${columnsToInsert[0].table}_${i}`,
+              ];
+            }
+          }
+        });
+
+        const formattedSQL = format(insertSeedSQL);
+        const finalSQL = formattedSQL.replace(/@ /g, '@').replace(/SET\s\s+/g, 'SET ');
+        console.log(finalSQL);
+        // console.log(chalk.green`Succesfully created \n${targetFilePath}`);
```
Comment (kirakoki) [inline cli/recipes/seed/index.js:288]: good idea to assure the user when the seeds are done.

**Review by kirakoki — APPROVED** [review]

File: cli/recipes/seed/utils/getFakerExample.js (hunk 1 above)
Comment (ersel) [reply cli/recipes/seed/utils/getFakerExample.js:31]: uncommented these lines now ✅

**Pull request merged**